### PR TITLE
tests(devtools): ensure WebSQL table is populated

### PR DESCRIPTION
**Pull request by adamraine**
Should prevent the flakiness in DevTools tests.

### CODE REVIEW

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
   await TestRunner.loadTestModule('lighthouse_test_runner');
   await TestRunner.showPanel('lighthouse');
+  await TestRunner.RuntimeAgent.invoke_evaluate({
+    expression: 'webSqlPromise',
+    awaitPromise: true,
+  });
```
Comment (connorjclark) [inline third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: I tried fixing this too. I did this:

image

I was trying to make LH wait for the DB by waiting for first paint.

But... this went from flaky to never seeing the DB usage. I never figured out why.

Comment (connorjclark) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: _ooooh_, I see what you've done. You made the DB get initiated before LH even runs.

So how was this ever working before? :)

Comment (adamraine) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: > So how was this ever working before?

It was flaky, sometimes the row would be added to the table in time, and sometimes it wouldn't. In CI it usually got added in time so I just kinda ignored the problem when it showed up locally.

Comment (adamraine) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: > To make that more clear can you move this to right after the navigate?

Move what to the right? Sorry don't understand.

**Review by connorjclark — APPROVED** [review]

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js (hunk 1 above)
Comment (connorjclark) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: Not "to the right". "right after" `TestRunner.navigatePromise`.

Comment (adamraine) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js:14]: 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃

**Pull request merged**